libs/utils/utils.js

```diff
@@ -763,9 +763,10 @@ async function decorateIcons(area, config) {
   if (icons.length === 0) return;
   const { base } = config;
   loadStyle(`${base}/features/icons/icons.css`);
-  loadLink(`${base}/img/icons/icons.svg`, { rel: 'preload', as: 'fetch', crossorigin: 'anonymous' });
-  const { default: loadIcons } = await import('../features/icons/icons.js');
-  await loadIcons(icons, config);
+  loadLink(`${base}/utils/federated.js`, { rel: 'preload', as: 'fetch', crossorigin: 'anonymous' });
```

Comment:
This currently has a different mime/type leading to two requests.

<img width="1371" alt="image" src="https://github.com/user-attachments/assets/8420ee55-2028-47b5-89a1-eea24ce542e5">

```suggestion
  loadLink(`${base}/utils/federated.js`, { rel: 'preload', as: 'script', crossorigin: 'anonymous' });
```

Reply:
Ahh. I missed that part. I've updated this and it only gets called 1 time now. Thanks